test.runsettings

```diff
@@ -9,4 +9,20 @@
       -->
       <TestTimeout>660000</TestTimeout>
    </MSTest>
+	<DataCollectionRunSettings>
+	  <DataCollectors>
+		<DataCollector friendlyName="Code Coverage" uri="datacollector://Microsoft/CodeCoverage/2.0" assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+			<Configuration>
+				<CodeCoverage>
+					<ModulePaths>
+					  <Exclude>
+						<ModulePath>.*unittests\.dll$</ModulePath>
```

Comment:
```suggestion
						<ModulePath>.*tests\.dll$</ModulePath>
```

Reply:
+1. We want to exclude e2e tests as well, after all